projects/solv-protocol-funds/index.js

```diff
@@ -103,9 +104,84 @@ async function mantleTvl(ts, _, _1, { api }) {
     api.add(klp["address"], amount)
   })
 
+  await iziswap(ts, _, _1, { api });
+
   return api.getBalances()
 }
 
+async function iziswap(ts, _, _1, { api }) {
```

Comment:
I should create a helper (like the one for uniswap v3 NFTs) at some point so people dont have to create this each time.

Reply:
Yes, I think this is best too.